CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file
 automatically by Versionist. DO NOT EDIT THIS FILE MANUALLY!
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+# v.0.1.1
```

Comment:
Why is this edit in here? It should not be edited manually, as it is created by versionbot.

Reply:
Agreed!